packages/admin-panel-server/src/app/createApp.ts

```diff
@@ -47,6 +47,7 @@ export function createApp() {
   const app = new OrchestratorApiBuilder(new TupaiaDatabase())
     .useSessionModel(AdminPanelSessionModel)
     .verifyLogin(hasTupaiaAdminPanelAccess)
+    .get('/v1/user', handleWith(UserRoute))
```

Comment:
I'm thinking that since this is admin-panel-server now instead of viz-builder-server, we might have to be more specific with our endpoints path. Maybe we can have `/v1/vizBuilder/user` (and similar pattern for other viz-builder endpoints as well). But I think you don't need to change this now, I can bulk change that when it is decided

Reply:
Yep for sure. If you end up adding that middleware for the permission checking then we might be able to remove this route from admin-panel-server altogether and use the '/me' endpoint from meditrak-server. I really just needed it to add the isBESAdmin boolean. Again feel free to refactor or delete this down the track - I was mostly just getting something working so I could unblock my front-end work.